src/bentoml/_internal/runner/runnable.py

```diff
@@ -112,9 +112,12 @@ def method(
         def method_decorator(
             meth: t.Callable[t.Concatenate[T, P], R]
         ) -> RunnableMethod[T, P, R]:
+            import inspect
+
             return RunnableMethod(
                 meth,
                 RunnableMethodConfig(
+                    is_stream=inspect.isasyncgenfunction(meth),
```

Comment:
```suggestion
                    is_stream=inspect.isasyncgenfunction(meth) or inspect.isgeneratorfunction(meth),
```
The user defined function can be either sync or async

Reply:
Do you think we should enforce user to write an async function?

Reply:
No, since it isn't a big deal handling both on our side, we can have more flexibility.